kernelci/cli/job.py

```diff
@@ -27,25 +27,17 @@ def kci_job():
 
 @kci_job.command(secrets=True)
 @click.argument('name')
-@click.option('--node-id')
-@click.option('--node-json')
+@click.argument('input_node_id')
 @Args.config
 @Args.api
 @Args.indent
 @catch_error
-def new(name, node_id, node_json, config,  # pylint: disable=too-many-arguments
+def new(name, input_node_id, config,  # pylint: disable=too-many-arguments
         api, indent, secrets):
     """Create a new job node"""
     configs = kernelci.config.load(config)
     helper = get_api_helper(configs, api, secrets)
-    if node_id:
-        input_node = helper.api.node.get(node_id)
-    elif node_json:
-        input_node = helper.load_json(node_json)
-    else:
-        raise click.ClickException(
-            "Either --node-id or --node-json is required."
-        )
+    input_node = helper.api.node.get(input_node_id)
```

Comment:
Although this isn't related to this PR, I'd appreciate it if you could add a check here for node existence to save another PR on this.
```suggestion
    input_node = helper.api.node.get(input_node_id)
    if not input_node:
        raise click.ClickException("Node not found with the provided ID")
```
Otherwise, we'll end up with the below error:
```
  File "/home/jeny/kernelCI/kernelci-core_env/src/kernelci-core/kernelci/cli/job.py", line 54, in new
    job_node = helper.create_job_node(job_config, input_node)
  File "/home/jeny/kernelCI/kernelci-core_env/src/kernelci-core/kernelci/api/helper.py", line 110, in create_job_node
    'parent': input_node['id'],
TypeError: 'NoneType' object is not subscriptable
```